SofaKernel/framework/sofa/helper/system/FileRepository.cpp

```diff
@@ -127,26 +127,16 @@ FileRepository::~FileRepository()
 {
 }
 
-std::string FileRepository::cleanPath( const std::string& path )
+std::string FileRepository::cleanPath(const std::string& path)
 {
-    std::string p = path;
-    size_t pos = p.find("//");
-    size_t len = p.length();
-    while( pos != std::string::npos )
-    {
-        if ( pos == (len-2))
-            p.replace( pos, 2, "");
-        else
-            p.replace(pos,2,"/");
-        pos = p.find("//");
-    }
-    return p;
+    msg_deprecated("FileRepository::cleanPath") << "Use FileSystem::cleanPath instead.";
```

Comment:
I think you should use dmsg_deprecated instead of msg_deprecated
and also use 
with https://en.cppreference.com/w/cpp/language/attributes/deprecated